### DISCUSSION TITLE
Hayward heater compatible?

### DISCUSSION THREAD

**Discussion by magimat** [aqualinkd/AqualinkD#169] [General]
Hi,

With the upcoming pool season, I am looking into some new automation project and trying to figure if my equipment would be compatible and I'm wondering if my actual pool heater is even compatible with all that,

I understand I would need to connect and use some rs485 protocol, but I'm not sure if I can do that with my equipment.

My heater is an Hayward HP50A and here is the manual for it:

https://www.hayward-pool.ca/assets/documents/pools/pdf/manuals/can-HP40A-HP50A-HP80A-bilingual.pdf

page 22 shows the wiring diagram for my model and there is come controller unit identified, but I'm not sure it is the right thing.

Anyway has any experience with this heater model and could point me in some direction here?

thanks!

**Comment by sfeakes** [comment]
If you can control the heater from control panel/ keypad, (assuming Jandy control panel), then AqualinkD can control it.  Most heaters don’t use the RS485 protocol anyway. They are usually set to max heat wired through a relay and the control panel cycles them on/off once up to temperature.
